ssl-tool/openssl.cnf

```diff
@@ -90,6 +90,17 @@ keyUsage = critical, digitalSignature, keyEncipherment
 extendedKeyUsage = serverAuth, clientAuth
 subjectAltName = @alt_names
 
+[ server_cert ]
```

Comment:
This may break existing certificate generation process: https://github.com/Alfresco/alfresco-ssl-generator/pull/18